Tests/HyperwalletErrorTests.swift

```diff
@@ -81,11 +81,11 @@ class HyperwalletErrorTests: XCTestCase {
     func testHyperwalletError_generalHttpError() {
         let hyperwalletError = HyperwalletError(message: "Please check your login credentials and try again",
                                                 code: "INCORRECT_LOGIN_CREDENTIALS")
```

Comment:
This still qualifies for 401. Use 403 Use some other message like, "The caller does not have access to the requested resource" and code "FORBIDDEN"

Reply:
Changed to FORBIDDEN